contrib/endpoints/src/api_manager/path_matcher.h

```diff
@@ -120,10 +124,10 @@ class PathMatcherBuilder {
   // be multiple templates in different services on a server. Consider moving
   // this to PathMatcherNode.
   std::set<std::string> custom_verbs_;
-  typedef PathMatcher::MethodData MethodData;
+  typedef typename PathMatcher<Method>::MethodData MethodData;
```

Comment:
Since compiler would complain without it :)

See http://stackoverflow.com/questions/18385418/c-meaning-of-a-statement-combining-typedef-and-typename